float64 x_position
float64 y_position
float64 z_position
---
float64 joint1
float64 joint2
float64 joint3